app/views/restaurant-detail.html

```diff
@@ -67,8 +67,8 @@ <h1 class="page-header">餐厅信息管理</h1>
                                        ng-model="basicInfo.restaurantAddress"/>
                             </td>
                             <td width="25%">
-                                餐厅经纬度:<span class="explain" style="padding-left: 10px"><a target="_blank" href="http://api.map.baidu.com/lbsapi/getpoint/index.html">经纬度怎么填?点我查询</a ></span>
-                                <input class="form-control" ng-model="longitudeNLatitude" placeholder="格式: 00.00000,11.11111">
+                                餐厅经纬度<span class="explain" style="padding-left: 10px"><a target="_blank" href="http://api.map.baidu.com/lbsapi/getpoint/index.html">经纬度怎么填？点我查询</a ></span>
```

Comment:
I purposely delete it because i did not see a colon in other table cells such as:

```
<td width="25%">餐厅电话<span class="explain" style="padding-left: 10px">若有多个电话号码，用一个空格分开</span>
```

Anyhow, correct it if I am wrong with it.